rwiki/rwiki-impl/impl/src/java/uk/ac/cam/caret/sakai/rwiki/component/service/impl/RWikiObjectServiceImpl.java

```diff
@@ -1026,9 +1023,14 @@ public String archive(String siteId, Document doc, Stack stack,
 		{
 			String defaultRealm = siteService.getSite(siteId).getReference();
 
-			wikiSecurityService
-					.checkAdminPermission(RWikiObjectService.REFERENCE_ROOT
-							+ defaultRealm);
+			// Todo, do we need to check for permissions here?
```

Comment:
Delete this.  You don't need this.  Non site.upd users won't ever seen an "Export" button anyways - and if they somehow sneaked by the archive would fail badly long before it got to wiki.